src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -127,13 +127,13 @@ private static String getGitBlameResult(RepoConfiguration config, String filePat
     }
 
     /**
-     * Returns true if the first line in the file at {@code repoRoot}'s {@code relativePath} contains the reused tag.
+     * Returns true if file at {@code repoRoot}'s {@code relativePath} has 0 lines.
      */
-    private static boolean isReused(String repoRoot, String relativePath) {
+    private static boolean isEmptyFile(String repoRoot, String relativePath) {
         Path path = Paths.get(repoRoot, relativePath);
         try (BufferedReader br = new BufferedReader(new FileReader(path.toFile()))) {
             String firstLine = br.readLine();
-            if (firstLine == null || firstLine.contains(REUSED_TAG)) {
```

Comment:
Instead of reading the first line of the file and checking if the `BufferedReader` returns null, should we use other simpler methods of verifying if the file is empty?

For example, we could use the `length` method of `java.io.File`.
Please see [the java docs here](https://docs.oracle.com/javase/8/docs/api/java/io/File.html#length--)

Reply:
Yes, I agree with that. We should add a method in FileUtil to check.

Reply:
Done.